hello-world/src/activities.ts

```diff
@@ -1,5 +1,6 @@
 // @@@SNIPSTART typescript-hello-activity
 export async function greet(name: string): Promise<string> {
+  console.log('ACTIVITY LOG', name);
```

Comment:
This goes as a snippet into our docs, let's remove the console.log

Reply:
oof this shouldnt have been in there at all. removing